pkg/conversion/doc.go

```diff
@@ -14,6 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// +k8s:deepcopy-gen=generate
```

Comment:
I think we don't want this one. I think that conversion was in input dirs to workaround something previously (not to really generate any functions).